src/js/controls/Header.js

```diff
@@ -109,9 +181,9 @@ lm.utils.copy( lm.controls.Header.prototype, {
 	 * @param {lm.item.AbstractContentItem} contentItem
 	 */
 	setActiveContentItem: function( contentItem ) {
-		var i, j, isActive, activeTab;
+		var isActive;
 
-		for( i = 0; i < this.tabs.length; i++ ) {
+		for(var i = 0; i < this.tabs.length; i++ ) {
```

Comment:
So you're now declaring this i variable here, but you should actually use `let` to make it a block scoped variable: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Using `var` always declares the variable in a global context. So you can do some crazy stuff like this:
```
if (true) {
  var foo = 'this is foo';
  for (var i = 0; i < 10; i++) {
    var foo = 'foo is now ' + i;
  }
}
console.log('foo=', foo, '\ni=', i)
```

Which outputs:
```
foo= foo is now 9 
i= 10
```

Reply:
It's still scoped to the setActiveContent function as it was before, not global. Also we don't have babel here, so I didn't want to go all modern.